tests/layer_tests/pytorch_tests/test_squeeze.py

```diff
@@ -28,8 +29,8 @@ def forward(self, x):
 
         return aten_squeeze(dim), ref_net, "aten::squeeze"
 
-    @pytest.mark.parametrize("dim,dynamic_shapes", [(-2, True), (0, True), (None, False)])
+    @pytest.mark.parametrize("dim", [-2, -1, 0, None, [-1, -2], 1, 2])
     @pytest.mark.nightly
     @pytest.mark.precommit
-    def test_squeeze(self, dim, ie_device, precision, ir_version, dynamic_shapes):
-        self._test(*self.create_model(dim), ie_device, precision, ir_version, dynamic_shapes=dynamic_shapes)
+    def test_squeeze(self, dim, ie_device, precision, ir_version):
+        self._test(*self.create_model(dim), ie_device, precision, ir_version, dynamic_shapes=False)
```

Comment:
Why you switch off dynamic shapes? I think your approach shouldn't depend on knowing shapes, am I wrong?

Reply:
I was thinking that dynamic shapes flag just changes the shapes, not rank. Please leave a comment that Squeeze will fail with dynamic shapes

Reply:
Thanks, added comment to test. Should translation also contain this comment?